### PR TITLE
Manual: Change examples to addons.

### PR DESCRIPTION
**Pull request by LeviPesin**
Related issue: https://github.com/mrdoob/three.js/issues/24595
Follow-up to https://github.com/mrdoob/three.js/pull/24579

**Description**

Update manual for examples -> addons change.

### CODE REVIEW

File: manual/en/fundamentals.html [hunk 1]
```diff
@@ -422,12 +422,12 @@ <h3>es6 modules, three.js, and folder structure</h3>
 libraries they'll both reference the same <code class="notranslate" translate="no">three.module.js</code> file.
 </p>
 <pre class="prettyprint">import * as THREE from './someFolder/build/three.module.js';
-import {OrbitControls} from './someFolder/examples/jsm/controls/OrbitControls.js';
+import {OrbitControls} from './someFolder/addons/controls/OrbitControls.js';
```
Comment (LeviPesin) [inline manual/en/fundamentals.html:425]: This file needs a heavier rewriting

Comment (LeviPesin) [reply manual/en/fundamentals.html]: Something like this: https://github.com/mrdoob/three.js/pull/24888/commits/b6e279709ee57d0a4b3707e82d7d0ceede8276a5

Comment (LeviPesin) [reply manual/en/fundamentals.html]: Possibly something about Node/NPM usage also should be added there.

**Comment by greggman** [comment]
lgtm

**Comment by LeviPesin** [comment]
Note that other languages' installation.html and prerequisites.html should also be changed.

**Pull request merged**